src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -286,6 +363,7 @@ def handle_notification(self, request):
                 last_numbers=card_pan[-4:],  # last 4 digits
                 owner=order.owner,
                 token=card_token,
+                initial_issuer_transaction_identifier=initial_issuer_transaction_identifier,
```

Comment:
According to my comment above, about tokenize an already registered credit card, I think here should do an `update_or_create`

Reply:
I'm not sure to get all this tokenize an already registered credit card. Isn't a registered card already tokenized ?

Reply:
I tried to explain better my thought here. https://github.com/openfun/joanie/pull/766#discussion_r1601722209